JM/CF/Scripts/3_Game/CommunityFramework/MVC/Controller.c

```diff
@@ -0,0 +1,147 @@
+
+
+
+class Controller: MVCEventHandler
+{
+	// Private members
+	private ref PropertyHashMap m_ControllerPropertyHashMap;
+	
+		
+	protected ref DataBindingHashMap m_ViewBindingHashMap = new DataBindingHashMap();
+	protected ref PropertyHashMap m_PropertyHashMap = PropertyHashMap.FromType(Type());
+	typename GetPropertyType(string property_name) {
```

Comment:
Please follow code formatting

Reply:
Only Getters and Setters use 1-line bracket formatting. Keeps things much cleaner and makes them easier to differentiate from proper methods